src/main.cpp

```
@@ -166,6 +166,10 @@ int main(int argc, char** argv) {
     if (!list) {
         try {
             raw_progs = elf.get_programs(desired_section, desired_program);
+        } catch (const InternalError& e) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)